src/VisualStudio/ProjectSystem/Microsoft.VisualStudio.ProjectSystem.CSharp.UnitTests/ProjectSystem/Designers/PropertiesFolderProjectTreeModifierTests.cs

```diff
@@ -0,0 +1,448 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.ProjectSystem.Designers.Imaging;
+using Microsoft.VisualStudio.Testing;
+using Xunit;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Designers
+{
+    [UnitTestTrait]
```

Comment:
It groups the project system tests under single node in Test Explorer for easy run, I'll rename it to make sure it says something regarding project system (https://github.com/dotnet/roslyn/issues/6772)